im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+        </group>
+      </if>
+      <if match="none" variable="title">
+        <group>
+          <group>
+            <text variable="collection-title" font-style="italic"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=" ."/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text macro="volume" prefix=", "/>
+          <text macro="editor"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if match="any" variable="container-title">
+        <text macro="editor" prefix=" "/>
+        <group prefix=" (" suffix=") ">
+          <group>
+            <text variable="collection-title"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=". "/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text variable="volume" prefix=", Bd. "/>
+        </group>
+      </if>
+      <if match="none" variable="container-title">
+        <group>
+          <text variable="collection-title" font-style="italic" prefix=" " suffix="."/>
+          <text macro="collection-editor" prefix=" "/>
+          <text macro="volume" prefix=" "/>
+          <text macro="editor" font-style="normal" prefix=" "/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition-number">
+    <choose>
+      <if match="any" variable="edition">
+        <text variable="edition" prefix=" " suffix=". "/>
+        <label plural="never" variable="edition"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-author">
+    <choose>
+      <if match="any" variable="container-author">
+        <text value="Ders.: "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="number-of-volumes">
+    <text variable="number-of-volumes" prefix=" von "/>
+    <label prefix=" " suffix=" " variable="number-of-volumes" form="short"/>
+  </macro>
+  <macro name="archive">
+    <text variable="archive" font-style="italic" prefix=" In: "/>
+    <text variable="publisher-place" prefix=" (" suffix=")"/>
+    <text variable="archive_location" prefix=", "/>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" collapse="year" after-collapse-delimiter=";">
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <choose>
+        <if match="any" position="ibid-with-locator">
+          <text term="ibid" suffix=" "/>
+          <text macro="citation-locator"/>
+        </if>
+        <else-if match="any" position="ibid">
+          <text term="ibid"/>
+        </else-if>
+        <else>
+          <group>
+            <choose>
+              <if match="any" variable="title-short">
+                <text variable="title-short"/>
```

Comment:
The idea is, that if there is a short title it should be used instead of (Author Year). So for example if the book "Ich und Du" by Martin Buber has its title also as its short title it should be cited inline as "(Ich und Du, S. 27)" instead of "(Buber 1996)".